x-pack/plugins/lists/server/routes/create_endpoint_list_route.ts

```diff
@@ -38,18 +37,15 @@ export const createEndpointListRoute = (router: IRouter): void => {
         // Our goal is be fast as possible and block the least amount of
```

Comment:
⛰️  cliffhanger! Block the least amount of what? 😄

Reply:
haha, ok there's enough here I will fix these issues in this PR and re-push. Thanks for looking over this carefully. Appreciate it.